config/jobs/image-pushing/k8s-staging-nfd-operator.yaml

```diff
@@ -0,0 +1,32 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes-sigs/node-feature-discovery-operator:
+    # The name should be changed to match the repo name above
+    - name: post-node-feature-discovery-operator-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        # This is the name of some testgrid dashboard to report to.
+        # If this is the first one for your sig, you may need to create one
+        testgrid-dashboards: sig-node-node-feature-discovery
+        testgrid-tab-name: push-image
```

Comment:
This is reserved for the operand. You need to use something else, e.g. `operator-push-image`

Reply:
makes sense now that we will be sharing with NFD

Reply:
updated